testsuite/testsuite-buildtool/pom.xml

```diff
@@ -42,11 +41,11 @@
       <artifactId>arquillian</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
+    <!-- <dependency>
       <groupId>org.jboss.arquillian.junit</groupId>
       <artifactId>arquillian-junit-container</artifactId>
       <scope>test</scope>
-    </dependency>
+    </dependency> -->
```

Comment:
Maybe remove instead of commenting out? Was removed in a lot of other modules too.